components/blocks/button.js

```diff
@@ -2,12 +2,20 @@ import Link from "next/link";
 
 import styles from "./button.module.css";
 
-const Button = ({ children, link }) => {
-  return (
-    <Link href={link}>
-      <button className={styles.Button}>{children}</button>
-    </Link>
-  );
+const Button = ({ children, link, target }) => {
+  if (target) {
+    return (
+      <a href={link} target={target}>
```

Comment:
Should we add `rel="noopener noreferrer"` when `target` is set to `_blank`? It was a best practice a while back to prevent cross-scripting, but I believe now [browsers handle it automatically](https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noopener)?

Reply:
All versions of major browsers now automatically use the behavior of rel="noopener" for any target="_blank" link, but I added it on b276d7c for best practice / convention, although it is redundant.